### PR TITLE
General: Optimized OCIO configs

### PR DESCRIPTION
**Pull request by antirotor**
## Description

OCIO configs vendorized as git submodule had a huge size, adding almost 3Gb to OpenPype.  This is changing it to use reduced [OCIO configs](pypeclub/OpenColorIO-Configs) managed by Poetry.

### Testing

This should build correctly and that darn `maketx` color space issue in Maya should be still fixed.

🔧 **This will go to 3.14 release**

### CODE REVIEW

**Review by Allan-I — COMMENTED** [review]
Code looks good to me, will test it first thing tomorrow!

**Comment by antirotor** [comment]
So Github has bandwidth limit on Git with LFS, so this is not option. We need to reimplement this to serve it as a zip file the same way as we are now doing with ffmpeg/oiio.

File: poetry.lock [hunk 1]
```diff
@@ -147,25 +147,25 @@ python-versions = ">=3.5"
...
 
 [[package]]
 name = "attrs"
-version = "21.4.0"
+version = "22.1.0"
 description = "Classes Without Boilerplate"
 category = "main"
 optional = false
-python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+python-versions = ">=3.5"
```
Comment (iLLiCiTiT) [inline poetry.lock:162]: I think we're still using `attrs` in Python 2 DCCs. We should maybe add it to python 2 vendor.

**Comment by Allan-I** [comment]
What do I need to do to test this out?  Just normally pull in and publish a look?

**Comment by iLLiCiTiT** [comment]
You should run `~/openpype/tools/create_env` script to update third party libs.

**Comment by Allan-I** [comment]
Thank you Kuba, this currently fails because the path evaluates to `C:\Users\Dumbledoor\Documents\Code\OpenPype\vendor\bin\ocioconfigOpenColorIOConfigs\nuke-default\config.ocio` which doesn't seem to exist.

File: openpype/hosts/maya/plugins/publish/extract_look.py [hunk 2]
```diff
@@ -40,11 +40,13 @@ def get_ocio_config_path(profile_folder):
...
         str: Path to vendorized config file.
     """
+
     return os.path.join(
         os.environ["OPENPYPE_ROOT"],
         "vendor",
-        "configs",
-        "OpenColorIO-Configs",
+        "bin",
+        "ocioconfig"
```
Comment (Allan-I) [inline openpype/hosts/maya/plugins/publish/extract_look.py:48]: "ocioconfig"`

Comment (Allan-I) [inline openpype/hosts/maya/plugins/publish/extract_look.py:48]: This is missing a comma I believe
from `"ocioconfig"` to `"ocioconfig",`

**Review by Allan-I — CHANGES_REQUESTED** [review]

File: openpype/hosts/maya/plugins/publish/extract_look.py (hunk 2 above)
Comment (iLLiCiTiT) [reply openpype/hosts/maya/plugins/publish/extract_look.py:48]:
```suggestion
        "ocioconfig",
```

**Review by Allan-I — APPROVED** [review]
Tested and works!

**Pull request merged**